### PR TITLE
Board: Add TableView

### LINKED ISSUE #1744: Board: Table/List view
Board: Table/List view

Similar to

### PR DESCRIPTION
**Pull request by Dvinyanin**
Signed-off-by: Dvinyanin Alexandr <dvinyanin.alexandr@gmail.com
Screenshot from 2022-05-16 14-14-15
Screenshot from 2022-05-16 14-14-17
Screenshot from 2022-05-16 19-53-50
>

### CODE REVIEW

File: plugins/board-resources/src/components/editor/CardDetails.svelte [hunk 1]
```diff
@@ -85,17 +52,12 @@
...
       <div class="text-md font-medium">
         <Label label={board.string.Members} />
       </div>
-      <div class="flex-row-center flex-gap-1">
-        {#each members as member}
-          <MemberPresenter value={member} size="large" menuItems={getMenuItems(member)} />
-        {/each}
-        <Button icon={IconAdd} shape="circle" kind="no-border" size="large" on:click={membersHandler} />
-      </div>
+      <MembersPresenter object={value} {membersHandler} />
```
Comment (annano) [inline plugins/board-resources/src/components/editor/CardDetails.svelte]: Can we revert these changes and use `UserBoxList` instead, like here https://github.com/hcengineering/anticrm/blob/main/plugins/board-resources/src/components/KanbanCard.svelte#L198

Comment (Dvinyanin) [reply plugins/board-resources/src/components/editor/CardDetails.svelte]: Can I change `items` to `value` or it's better to use proxy?
Screenshot from 2022-05-16 23-04-22

**Review by haiodo — APPROVED** [review]
